build-system/babel-plugins/babel-plugin-transform-rewrite-imports/index.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const rewrites = {
+  __proto__: null,
+  'preact': 'preact/dist/preact.module.js',
+  'preact/hooks': 'preact/hooks/dist/hooks.module.js',
+  'dompurify': 'dompurify/dist/purify.es',
+  'promise-pjs': 'promise-pjs/promise.mjs',
+  '@ampproject/worker-dom': '@ampproject/worker-dom/dist/amp/main.mjs',
+  '@ampproject/animations': '@ampproject/animations/dist/animations.mjs',
+};
+
+// Rewrites import paths from a colloquial name into a full name.
+//
+// Before:
+// import {createElement} from 'preact';
+// import {useState} from 'preact/hooks';
+//
+// After:
+// import {createElement} from 'preact/dist/preact.module.js';
+// import {useState} from 'preact/hooks/dist/hooks.module.js';
```

Comment:
Why not use block commenting style here?

Reply:
Eslint yelled at me when I tried. It wants JSDoc, but I'm not going to add typeinfo here.